pruner/find.go

```diff
@@ -10,7 +10,7 @@ import (
 // maxHeadersPerLoop is the maximum number of headers to fetch
 // for a prune loop (prevents fetching too many headers at a
 // time for nodes that have a large number of pruneable headers).
-var maxHeadersPerLoop = uint64(512)
```

Comment:
maybe const then?

Reply:
We change it's value in test. I don't have a good reason to redesign logic around this variable now. Unless someone is also against unexported global variable.